cars/v1/x_pack/README.md

```diff
@@ -1,18 +1,29 @@
-This directory contains example configurations for x-pack:
+# This directory contains example configurations for x-pack:
 
 * `security`: Configures TLS for all HTTP and transport communication using self-signed certificates.
-* `monitoring-local`: Enables x-pack monitoring with export to the current cluster.
+* `monitoring`: Enables x-pack monitoring with export to the current cluster.
```

Comment:
Should this be `monitoring-http` and the description should be changed in the sense that it exports to a different cluster? Also, `ml` seems to be missing.